aspnet/migration/aspnetcoremigration.rst

```diff
@@ -0,0 +1,190 @@
+Migrating from ASP.NET to ASP.NET Core
+======================================
+
+In this article:
+
+.. contents:: Sections
+  :local:
+  :depth: 1  
+
+Overview
+--------
+
+This migration guide will cover the essentials on how to migrate projects from ASP.NET to ASP.NET Core. 
+
+For information on migrating from DNX to .NET CLI, see the `DNX RC1->RC2 migration guide <http://placeholder>`_. 
+
+For information regarding a complete list of breaking changes in RC2 for ASP.NET, see the `ASP.NET Announcements page <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0-rc2>`_
+
+Implement main with WebHostBuilder logic
+----------------------------------------
+
+Since ASP.NET core apps are just console apps, you must explicitly call ``Main`` to build up the host and tell it to start listening.
```

Comment:
you must define an entry point for your application (``Program.Main``) that sets up a web host and tells it to start listening